### ISSUE TITLE
Cell formula dialog sizing.

### ISSUE THREAD

**Issue by nashnet** [nashnet/manhattan-issues#70]
@esaruoho - Can the Cell Code dialog be resizable?
(reason being is that Workshops -> Bytes + Pieces -> Dynamics Cell Code for 02:Bass is larger than what the dialog shows.

**Comment by nashnet** [comment]
It should expand automatically to best fit the code (width and height) - try scrolling the pattern down so that the dialog is at the top of the screen? Ironic that this is the only issue you report with out a screenshot! 🤣

**Comment by esaruoho** [comment]
took me a while to find it, glad i wrote "Cell Code" there. anyway, here's the top, here's the bottom of it.
<img width="684" alt="Screenshot 2023-10-04 at 23 53 39" src="https://github.com/nashnet/manhattan-issues/assets/4966687/5bb4a3ca-fd87-483a-ab39-40f405bff1e5">
<img width="654" alt="Screenshot 2023-10-04 at 23 53 53" src="https://github.com/nashnet/manhattan-issues/assets/4966687/8906a755-0c5f-4e78-9f43-7c94f37ebbd8">

basically the Cell Code has so much stuff (and there's no scrollbar, and scrolling with touch gestures within the Cell Code) that not all of it gets shown in the dialog itself. you need to click on it and use the cursor down keys. @nashnet i hope this helps!

**Comment by nashnet** [comment]
Code editor now can resize a little bigger, but will also show scrollbars when code is bigger than available space - and mouse wheel (or trackpad gestures) should scroll code when positioned over code, or pattern if not.

**Comment by esaruoho** [comment]
awesome, thank you very much!
